tools/syz-kconf/parser.go

```diff
@@ -0,0 +1,284 @@
+// Copyright 2020 syzkaller project authors. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+package main
```

Comment:
Maybe some comment about what this does? Or is it documented in a .md somewhere?

Reply:
There is README.md with more details here:
https://github.com/google/syzkaller/pull/1975/commits/d5b2aa4a509ffb7cded9d63100216eb3ead3a201

I've added a short package comment to point to that README.md.